src/app/person/person.cpp

```diff
@@ -106,3 +116,19 @@ bool Person::validateControlNumber(const std::string& pesel) const
     }
     return true;
 }
+
+bool Person::validateGender() const
+{
+    std::string gender = pesel_.substr(9, 1);
+
+    if (std::stoi(gender) % 2 == 1 and gender_ == Gender::Male)
+    {
+        return true;
+    }
+    else if (std::stoi(gender) % 2 == 0 and gender_ == Gender::Female)
+    {
+        return true;
+    }
+    else
+        return false;
+}
```

Comment:
I think we should perform validation on pass-by value and then set it in the object. 
Please use parameter instead value from the object

Reply:
I agree. In general all validation should happen before setting pesel. We don't want to set incorrect value and only then realize that it was wrong.

Reply:
Done